knip.json

```
@@ -1,6 +1,6 @@
 {
 	"$schema": "https://unpkg.com/knip@5.61.2/schema.json",
-	"entry": ["src/**/*.test.*", "src/index.ts"],
+	"entry": ["src/**/*.test.*"],
```

This is out of scope for this change. We will remove it in a follow up.
```suggestion
	"entry": ["src/**/*.test.*", "src/index.ts"],
```